lib/src/upgrader.dart

```diff
@@ -277,19 +277,29 @@ class Upgrader {
         var count = appcast.items == null ? 0 : appcast.items!.length;
         print('upgrader: appcast item count: $count');
       }
+      final criticalUpdateItem = appcast.criticalUpdateItem();
       final bestItem = appcast.bestItem();
       if (bestItem != null &&
           bestItem.versionString != null &&
           bestItem.versionString!.isNotEmpty) {
         if (debugLogging) {
           print(
               'upgrader: appcast best item version: ${bestItem.versionString}');
+          print(
+              'upgrader: appcast critical update item version: ${criticalUpdateItem?.versionString}');
```

Comment:
What does `criticalUpdateItem?.versionString` print when `criticalUpdateItem` is null?

Reply:
If `criticalUpdateItem` is null, `criticalUpdateItem?.versionString` would print null. I leave it as is as this is for debugging anyway.